screens/AllTasks.tsx

```diff
@@ -1,15 +1,14 @@
-import React, { useEffect, useState } from "react";
+import React, { useContext, useEffect, useState } from "react";
 import { Text, Button, View, FlatList } from "react-native";
 import TaskList from "../components/TaskList";
 import { useSQLiteContext } from "expo-sqlite";
-import { Task } from "../models/tasks";
 
-interface Cat {
+type Cat = {
```

Comment:
Improve naming of types, never use a bad name to a variable when a better word extsists and describes variable better.